Automattic/wp-calypso
# Store: Remove prices object from plans and products

#### Changes proposed in this Pull Request

This removes the `prices` property from plans and products as its data is no longer accurate and it will soon be removed from the APIs. For the most part the changes are only in tests, as the property should not actually be used anywhere currently. The only non-test change in this PR is removing `prices` from the state schema files for `products-list` and `plans`.

See pbOQVh-ms-p2

#### Testing instructions

From what I can tell, this property is not used anywhere in calypso, so there should be no testing necessary. However, for the sake of safety, we can test the parts of calypso that I think most heavily deal with prices: the plans page and the domain search page.

- Visit `/plans` for your site and verify that the prices all look accurate.
- Visit `/domains/add` for your site, then search for a domain and verify the prices all look accurate.

Change your currency to a different one and re-test the above to verify that they continue to work as expected.
